src/webview/main.ts

```
@@ -1,74 +1,32 @@
-    CLEAR: 'clear',
-    EXIT: 'exit',
-    SPLIT: 'split',
-    TERMINAL_CREATED: 'terminalCreated',
-    TERMINAL_REMOVED: 'terminalRemoved',
-    SWITCH_TERMINAL: 'switchTerminal',
-  },
-};
+import { WEBVIEW_TERMINAL_CONSTANTS, SPLIT_CONSTANTS } from './constants/webview';
+import { getWebviewTheme } from './utils/WebviewThemeUtils';
+import { SimpleStatusManager } from './managers/SimpleStatusManager';
+import { SplitManager, type TerminalInstance } from './managers/SplitManager';
```

ESLint

[error] 'TerminalInstance' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)